components/user-functionality-mgt/org.wso2.carbon.identity.user.functionality.mgt/src/main/java/org/wso2/carbon/identity/user/functionality/mgt/model/FunctionalityLockStatus.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.user.functionality.mgt.model;
+
+/**
+ * A data model class to define the functionality lock status element.
+ */
+public class FunctionalityLockStatus {
+
+    public static final FunctionalityLockStatus UNLOCKED_STATUS = new FunctionalityLockStatus(false, 0, null, null);
```

Comment:
Why do we need this static variable?

Reply:
Decided to have a static FunctionalityLockStatus object since we could use that whenever a functionality got unlocked instead of creating a new  FunctionalityLockStatus object.